examples/kitti/detectnet_solver.prototxt

```diff
@@ -0,0 +1,22 @@
+#
+net: "examples/kitti/detectnet_train_test.prototxt"
+
+max_iter: 32000
+display: 2
+test_iter: 185
+test_interval: 1228
+
+solver_type: ADAM
+lr_policy: "sigmoid"
```

Comment:
should not publicize sigmoid. move to step-wise, i think would be better to not release this interesting LR schedule